app/models/concerns/revivable.rb

```diff
@@ -12,7 +12,7 @@ def dormant?
   end
 
   def revivable?
-    activated? && last_auth_at < Rails.configuration.x.auth.dormant_account_threshold.ago
+    activated? && last_auth_at.present? && last_auth_at < Rails.configuration.x.auth.dormant_account_threshold.ago
```

Comment:
Added the last_auth_at.present? check to prevent nil exceptions, especially during testing.